app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -52,7 +52,6 @@ Get one step closer to the classroom with guidance tailored to you, including ho
 An adviser with years of teaching experience can help you if you're (one of the following):
 
 * ready to get into teaching
-* returning to teach at a secondary school in England and already have qualified teacher status (QTS)
```

Comment:
does this sentence need removing? even though we're removing returners content from GIT, they're still eligible for an adviser, right? @philspray

Reply:
I think we remove it - the blog is about getting ready to **apply**